### ISSUE TITLE
Category field : RootCategory option : unable to save the root and error getting it from graphql

### ISSUE THREAD

**Issue by charlyecho** [verbb/formie#891]
**Description**

**Steps to reproduce**
1. create a category
2. create a formie category field 
3. Try to set a root category in field "settings" : the field is not saved
4. Get the rootCategory in graphql : error (below)

### Form settings
Please pick from the options below.

- Multi-page form:  No
- Submission Method: Ajax or Page Reload
- Client-side Validation: Yes or No
- Custom Form Templates: Yes
- Frontend : graphql/nuxt

**Additional info**
- Plugin version: 1.5.16
- Craft version: 3.7.42
- Multi-site: No

**Additional context**
Graphql query for example : 
`query search {
  formieForm(handle: "test") {
    title
    pages {
      rows {
        ... on RowType {
          fields {
						handle
            ... on Field_Categories {
              id
              name
              rootCategory
            }
          }
        }
      }
    }
  }
}`

result is : 

`{
  "errors": [
    {
      "message": "String cannot represent value: [{\"id\":\"198\",\"siteId\":\"1\"}]",
      "extensions": {
        "category": "graphql"
      }
    }
  ],
  "data": {
    "formieForm": {
      "title": "test",
      "pages": [
        {
          "rows": [
            {
              "fields": [
                {
                  "handle": "test",
                  "id": "180",
                  "name": "test",
                  "rootCategory": null
                }
              ]
            }
          ]
        }
      ]
    }
  }
}`

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.5.16",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.5.17](https://github.com/verbb/formie/releases/tag/1.5.17)
